index.js

```diff
@@ -1,5 +1,49 @@
 "use strict";
 
-module.exports.decryptHash = function () {
-	throw new Error("Not implemented");
-};
+var forge = require("node-forge");
+
+function trimZeroes(data) {
+	return data.toString().replace(/^\0+/, "").replace(/\0+$/, "");
+}
+
+function padKey(key) {
```

Comment:
there probably is, but I really want to have all of this code in one place and it's simple enough to not care about